<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>358</width>
    <height>354</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Быки и коровы</string>
  </property>
  <widget class="QPushButton" name="check_button">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>90</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Проверить!</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="input_line">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>90</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
   <property name="inputMask">
    <string>9999</string>
   </property>
  </widget>
  <widget class="QPushButton" name="start_game">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Новая игра</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="game_list">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>120</y>
     <width>256</width>
     <height>192</height>
    </rect>
   </property>
   <property name="columnCount">
    <number>2</number>
   </property>
   <column/>
   <column/>
  </widget>
  <widget class="QLabel" name="label_input">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Введите число</string>
   </property>
  </widget>
  <widget class="QPushButton" name="stat_button">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>30</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Рекорды</string>
   </property>
  </widget>
  <widget class="QLabel" name="game_status_label">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>30</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
